src/common/Log.hxx

```diff
@@ -0,0 +1,30 @@
+#ifndef LOG_H
+#define LOG_H
+#include <iostream>
+namespace ale
+{
+    class Logger
+    {
+    public:
+        enum mode{
+            Info = 0,
+            Warning = 1,
+            Error = 2
+        };
+        static void setMode(mode m);
```

Comment:
Please put a comment here describing the effect of changing the logger mode. Also, current_mode should probably be private.